extensions/mentions/js/src/forum/components/PostMentionedNotification.js

```diff
@@ -23,6 +23,8 @@ export default class PostMentionedNotification extends Notification {
   }
 
   excerpt() {
-    return truncate(this.attrs.notification.subject().contentPlain(), 200);
+    const contentPlain = this.attrs.notification.subject().contentPlain();
+
+    return contentPlain && truncate(contentPlain, 200);
```

Comment:
Would this be simpler, do you think? I don't want introduce any issues, like returning false, undefined, etc, and code not accounting for this.

```suggestion
    return truncate(this.attrs.notification.subject().contentPlain() || '', 200);
```

Reply:
Yup good thinking!